### PR TITLE
fix(svg viewer): cannot read prop 'container' of undefined

### LINKED ISSUE #377: TypeError: Cannot read property 'container' of undefined
Seems like an issue we want to fix

Sentry Issue: [OPERATION-SUPPORT-86K](https://sentry.io/organizations/cognite/issues/1090629332/?referrer=github_integration)

```
TypeError: Cannot read property 'container' of undefined
  at zoomOnCurrentAsset (../node_modules/@cognite/gearbox/dist/components/SVGViewer/index.js:447:1)
  at call (../node_modules/@cognite/gearbox/dist/components/SVGViewer/index.js:265:1)
  at step (../node_modules/@cognite/gearbox/dist/chunks/chunk-1c8e2aef.js:84:1)
  at next (../node_modules/@cognite/gearbox/dist/chunks/chunk-1c8e2aef.js:65:45)
  at fulfilled (../node_modules/@cognite/gearbox/dist/chunks/chunk-1c8e2aef.js:55:42)
```

### PR DESCRIPTION
**Pull request by mrtraser**
* fixes #377

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-465.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-465.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-465.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/SVGViewer.tsx [hunk 1]
```diff
@@ -508,7 +508,7 @@ export class SVGViewer extends React.Component<SvgViewerProps, SvgViewerState> {
 
   zoomOnCurrentAsset = (currentAsset: Element | null) => {
     const isDesktop = this.state.width > minDesktopWidth;
-    if (!currentAsset || !this.pinchZoomInstance.container) {
+    if (!currentAsset || !this.pinchZoomInstance) {
```
Comment (valerii-cognite) [inline src/components/SVGViewer/SVGViewer.tsx]: Won't line 524 `this.pinchZoomInstance.container.clientWidth` fail with type error if you are not checking `this.pinchZoomInstance.container` against `undefined`?

Comment (mrtraser) [reply src/components/SVGViewer/SVGViewer.tsx]: no, I guess, cause `PinchZoomInterface` interface describes `container` as non-optional property with `HTMLElement` type.

Comment (valerii-cognite) [reply src/components/SVGViewer/SVGViewer.tsx]: Good point as long as the type is correct

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-465.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-465.gearboxjs.preview.cogniteapp.com

**Pull request merged**